### PR TITLE
Added features and feature catogaries for DAS 3.1.0

### PR DESCRIPTION
**Pull request by Maninda**
Added features and feature categories for DAS 3.1.0 related features to Carbon feature p2 repository.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,6 +97,7 @@
         <carbon.dashboards.version.1.0.11>1.0.11</carbon.dashboards.version.1.0.11>
         <carbon.dashboards.version.1.0.18>1.0.18</carbon.dashboards.version.1.0.18>
         <carbon.dashboards.version.2.0.0>2.0.0</carbon.dashboards.version.2.0.0>
+		<carbon.dashboards.version.2.0.2>2.0.2</carbon.dashboards.version.2.0.2>
```
Comment (thusithathilina) [inline pom.xml]: formatting issues

File: pom.xml [hunk 2]
```diff
@@ -133,6 +137,7 @@
         <carbon.identity.version.5.1.2>5.1.2</carbon.identity.version.5.1.2>
         <carbon.identity.version.5.2.0>5.2.0</carbon.identity.version.5.2.0>
         <carbon.identity.version.5.0.8>5.0.8</carbon.identity.version.5.0.8>
+		<carbon.identity.saml2.version.5.1.2>5.1.2</carbon.identity.saml2.version.5.1.2>
```
Comment (thusithathilina) [inline pom.xml]: formatting issues

File: pom.xml (hunk 1 above)
Comment (Maninda) [reply pom.xml]: Corrected them.

**Review by thusithathilina — APPROVED** [review]

**Comment by thusithathilina** [comment]
I'm closing this PR since this has conflicts. New PR is https://github.com/wso2/carbon-feature-repository/pull/63

**Pull request closed without merging**